tests/ProxyManagerTest/Functional/RemoteObjectFunctionalTest.php

```diff
@@ -33,12 +36,15 @@
 final class RemoteObjectFunctionalTest extends TestCase
 {
     /**
-     * @param mixed   $expectedValue
+     * @param mixed $expectedValue
+     * @param string $method
      * @param mixed[] $params
+     * @return XmlRpcAdapter
      */
-    protected function getXmlRpcAdapter($expectedValue, string $method, array $params) : XmlRpcAdapter
+    protected function getXmlRpcAdapter($expectedValue, string $method, array $params): XmlRpcAdapter
     {
-        $client = $this->getMockBuilder(Client::class)->setMethods(['call'])->getMock();
+        /** @var Client|MockObject $client */
```

Comment:
This was just to stop annoyance from PHPStorm saying that we should pass `Zend\Server\Client` instead of `PHPUnit\Framework\MockObject\MockObject`. But I removed it now.

Reply:
Yeah, let's have as much type inference as possible, and type declarations only where psalm can't find a type :+1: